projects/storefrontlib/src/shared/utils/validators/custom-form-validators.ts

```diff
@@ -1,30 +1,79 @@
-import { AbstractControl, ValidationErrors } from '@angular/forms';
+import { AbstractControl, ValidationErrors, FormGroup } from '@angular/forms';
 import { EMAIL_PATTERN, PASSWORD_PATTERN } from '@spartacus/core';
 
 export class CustomFormValidators {
```

Comment:
Document breaking changes:

- removed emailDomainValidator and matchPassword

Reply:
Done